src/control/cmd/drpc_test/hello/drpc_test_module.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2018 Intel Corporation.
+// (C) Copyright 2018-2019 Intel Corporation.
```

Comment:
Id really like it if we didn't clutter PRs with minor fixes. The fact that we don't allow for mutliple logical commits in a PR makes things messy. If I was searching for drpc changes in the history I'd get this commit and then have to sort through it to figure out it actually has nothing to do with drpc changes.

Reply:
> Ideally you'd have some mechanism in place to ensure that only daos_server is the one calling this binary.

Well, that was the intent behind the setgid/setuid stuff in the RPM. On install, we create a new daos_admins group. daos_server's group is set to daos_admins and then the binary is setgid so that the group membership is assigned regardless of which user runs it. The daos_admin binary has its group also set to daos_admins and its permissions set to 4750, which allows root or daos_admins to execute it, but no one else.

With regard to using protobufs -- I just don't see the advantage, but I'm willing to be convinced otherwise. What is the benefit to adding another layer? The privileged binary uses the same req/res structs as the caller, so what do we gain by converting them into protobufs and then back to the same struct?